boards/xtensa/intel_adsp_cavs15/tools/lib/loglist.py

```diff
@@ -51,6 +51,6 @@ def print(self):
         for pair in sorted(self.loglist):
             if self.debug:
                 # Add slot number when debug is enabled
-                print('[{}] : {}'.format(*pair), end='')
+                print('{{[{}] : {}}};  '.format(*pair), end='')
```

Comment:
Not that this is a bad change, but FWIW: I remain on record that it would be better for all involved to replace this protocol with a simpler one than to try to add code to debug it.